components/FeatureHighlight/src/FontThemer/MDCFeatureHighlightFontThemer.h

```diff
@@ -42,6 +42,9 @@
  Learn more at docs/theming.md#migration-guide-themers-to-theming-extensions
  */
 + (void)applyFontScheme:(nonnull id<MDCFontScheme>)fontScheme
-    toFeatureHighlightView:(nonnull MDCFeatureHighlightView *)featureHighlightView;
+    toFeatureHighlightView:(nonnull MDCFeatureHighlightView *)featureHighlightView
```

Comment:
Do we need this since the class is deprecated?

Reply:
It's probably not entirely necessary but it's fine I think. I saw Jeff did it in a recent deprecation PR and I was like I'll do that too 🤷‍♂